crates/houston/src/profile/mod.rs

```diff
@@ -86,7 +86,8 @@ impl Profile {
     pub fn delete(name: &str) -> Result<(), HoustonProblem> {
         let dir = Profile::dir(name)?;
         tracing::debug!(dir = ?dir);
-        Ok(fs::remove_dir_all(dir)?)
+        Ok(fs::remove_dir_all(dir)
+            .map_err(|_| HoustonProblem::ProfileNotFound(name.to_string()))?)
```

Comment:
might want to actually make sure that this is always 100% the problem here, `remove_dir_all` is likely fallible in other ways

Reply:
Do you think this is something that need to be addressed now?

Reply:
meh - doesn't hurt. i fixed it up :smile: